client/layout/error.jsx

```diff
@@ -46,10 +46,7 @@ export function retry( chunkName ) {
 export function show( context, chunkName ) {
 	log( 'Chunk %s could not be loaded', chunkName );
 	bumpStat( 'calypso_chunk_error', chunkName );
-	context.store.dispatch( {
-		type: SECTION_SET,
-		section: false,
-	} );
+	context.store.dispatch( setSection( false, { section: false } ) );
```

Comment:
Storing booleans here seems really messy, but it seems it's not a new thing around here.

Reply:
Right, I think it's probably best to preserve this behaviour for the purposes of this PR.